ci-deploy/inside-container.sh

```diff
@@ -45,16 +46,20 @@ echo "$SERVER $SERVER_PUBLIC_KEY" > known_hosts
 # Sync, including deletes, but ignoring the commit-snapshots directory so we don't delete that.
 echo "Deploying build output..."
 rsync --rsh="ssh -o UserKnownHostsFile=known_hosts" \
-      --archive --compress --delete --verbose --exclude="commit-snapshots" \
+      --archive --chmod=D755,F644 --compress --verbose \
```

Comment:
Nit: I find the letter syntax much easier to understand than the bitmask numbers syntax. Would you be up for switching to that?

Reply:
I've never used that before, but from the man page I stitched together "u=rw,go=r" as the equivalent on 644, and "u=rwx,go=rx" for 755. Put together I think it'd be `--chmod="Du=rwx,Dgo=rx,Fu=rw,Fgo=r"`. Is that really how you'd write it?

Reply:
Well, it's at least clearer to me this way that some "D" user is getting read/write/execute; some "d" group is getting read/execute, some "F" user is getting read/write; and some "F" group is getting read. Before it was just opaque.

Reply:
Still hard to parse I guess, D and F are prefixes handled by rsync so that it only applies to directories and files, respectively, not the sames of anything.

The original problem I had in https://github.com/whatwg/resources.whatwg.org/pull/60, I think, was that after testing an rsync locally the result wasn't readable by the www-data user.

Another way to just ensure readability would be `--chmod=a+r,Da+x`, WDYT?

I still prefer the numbers by some margin since I'm so used to them and can tell exactly what the result will be, but ping @sideshowbarker for a second opinion.

Reply:
I also find the numbers more familiar. I agree the `D` and `F` are opaque, but the `--chmod=a+r,Da+x` has the same `D`…

Reply:
How about stick with the original thing but add a comment? @domenic?

Reply:
Sure, that works for me.